src/ctr_logging.c

```diff
@@ -54,6 +54,11 @@ static char *container_name = NULL;
 static char *container_tag = NULL;
 static size_t container_tag_len;
 
+typedef struct {
+	int iovcnt;
+	struct iovec iov[WRITEV_BUFFER_N_IOV];
+} writev_buffer_t;
+
```

Comment:
Just curious about this move?  I know writev_buffer_t is only used currently here in this c file and probably will only ever be used here.  But did you see a compilation or other error with it in the .h file?  The move is fine, just wondering about the motivation.

Reply:
no no compilation warning.  It is just to make explicit it is used only in this file and prevent in the future to have functions using write_buffer_t in other files

Reply:
Thanks for the 411 @giuseppe.